src/Illuminate/Console/Application.php

```diff
@@ -55,11 +55,13 @@ public function __construct(Container $laravel, Dispatcher $events, $version)
      */
     public function call($command, array $parameters = [])
     {
-        $parameters['command'] = $command;
+        $parameters = collect($parameters)->prepend($command);
 
         $this->lastOutput = new BufferedOutput;
 
-        return $this->find($command)->run(new ArrayInput($parameters), $this->lastOutput);
+        $this->setCatchExceptions(false);
```

Comment:
No i don't think so.

Its set to false because \Artisan::call() SHOULD throw exceptions generated from the command itself (as far as im aware, this is my understanding because there is no exception handling for its current state). because this is in the context  of calling it via other code, not via the console.

Because we are using the console application, it (as it should) has exception rendering turned on to display console friendly errors.

But when calling via the facade the exception should NOT get supressed (or rendered in console friendly output) as the calling code needs to receive those errors thrown.

For example

Command throws and exception, when running via:

```php artisan command```

artisan should catch exceptions and render them accordingly.

whereas:

```\Artisan::call('command', []);```

artisan should NOT catch the exception, the app should be responsible for it.

that is currently how it works.